src/ui/views/accountview.cpp

```diff
@@ -277,9 +277,9 @@ AccountView::AccountView(GtkWindow* parentWindow, AdwTabView* parentTabView, Gtk
     g_signal_connect(m_actExportAsCSV, "activate", G_CALLBACK((void (*)(GSimpleAction*, GVariant*, gpointer))[](GSimpleAction*, GVariant*, gpointer data) { reinterpret_cast<AccountView*>(data)->onExportAsCSV(); }), this);
     g_action_map_add_action(G_ACTION_MAP(m_actionMap), G_ACTION(m_actExportAsCSV));
     //Import from CSV Action
-    m_actImportFromCSV = g_simple_action_new("importFromCSV", nullptr);
-    g_signal_connect(m_actImportFromCSV, "activate", G_CALLBACK((void (*)(GSimpleAction*, GVariant*, gpointer))[](GSimpleAction*, GVariant*, gpointer data) { reinterpret_cast<AccountView*>(data)->onImportFromCSV(); }), this);
-    g_action_map_add_action(G_ACTION_MAP(m_actionMap), G_ACTION(m_actImportFromCSV));
+    m_actImportFromFile = g_simple_action_new("importFromFile", nullptr);
```

Comment:
Personally I would prefer to have 3 separate actions in the menu for each file type instead of making the user to open the list of filters in the dialog to see what formats can be imported/exported. @nlogozzo, what do you think?

Reply:
I thought about doing that at first, but I wanted to avoid as much code repetition as possible, and having three of them would have meant to have three different dialogs, three different functions... But if needed I can change

Reply:
You could use one dialog and one function, but change dialog's filter depending on what action was activated. But anyway, let's wait what Nick will say.

Reply:
I think the way it is now is good. Similar to how when you open a document on LibreOffice all the supported file types are available at once. 

Right now however there are 3 filters for each separate file type. I would add one filter above it all that says `Supported files` that has a pattern for each type of file.

```cpp
gtk_file_filter_set_name(filter, _("Supported files"));
gtk_file_filter_add_pattern(filter, "*.csv");
gtk_file_filter_add_pattern(filter, "*.ofx");
gtk_file_filter_add_pattern(filter, "*.qif");
gtk_file_chooser_add_filter(GTK_FILE_CHOOSER(openFileDialog), filter);
g_object_unref(filter);
```